<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1185</width>
    <height>677</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Учёт пациентов</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="foneTwo">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>210</y>
      <width>261</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(255, 237, 246)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="autorization">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>210</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(131, 6, 255);
font-size: 20px;</string>
    </property>
    <property name="text">
     <string>АВТОРИЗАЦИЯ</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="login">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>260</y>
      <width>241</width>
      <height>24</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Логин</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="infoTxt">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>1141</width>
      <height>581</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(207, 229, 255)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loginButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>340</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Войти</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="password">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>300</y>
      <width>241</width>
      <height>24</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Пароль</string>
    </property>
   </widget>
   <widget class="QLabel" name="fone">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1181</width>
      <height>631</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/rec/img/home.png</pixmap>
    </property>
   </widget>
   <zorder>fone</zorder>
   <zorder>infoTxt</zorder>
   <zorder>foneTwo</zorder>
   <zorder>autorization</zorder>
   <zorder>login</zorder>
   <zorder>loginButton</zorder>
   <zorder>password</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1185</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Программа</string>
    </property>
    <addaction name="admission"/>
    <addaction name="patientList"/>
    <addaction name="userInfo"/>
    <addaction name="createrInfo"/>
    <addaction name="exit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="admission">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/rec/img/login.png</normaloff>:/rec/img/login.png</iconset>
   </property>
   <property name="text">
    <string>Войти</string>
   </property>
  </action>
  <action name="patientList">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/rec/img/listPatient.png</normaloff>:/rec/img/listPatient.png</iconset>
   </property>
   <property name="text">
    <string>Список пациентов</string>
   </property>
  </action>
  <action name="userInfo">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/rec/img/infoClient.png</normaloff>:/rec/img/infoClient.png</iconset>
   </property>
   <property name="text">
    <string>Информация для пользователя</string>
   </property>
  </action>
  <action name="createrInfo">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/rec/img/infoCreater.png</normaloff>:/rec/img/infoCreater.png</iconset>
   </property>
   <property name="text">
    <string>Информация для разработчика</string>
   </property>
  </action>
  <action name="exit">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/rec/img/exit.png</normaloff>:/rec/img/exit.png</iconset>
   </property>
   <property name="text">
    <string>Выйти</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
